src/main/java/org/hyperledger/fabric/sdk/Endpoint.java

```diff
@@ -66,6 +66,11 @@
 import static org.hyperledger.fabric.sdk.helper.Utils.parseGrpcUrl;
 
 class Endpoint {
+    public static final String CLIENT_CERT_FILE = "clientCertFile";
+    public static final String CLIENT_KEY_FILE = "clientKeyFile";
+    public static final String CLIENT_CERT_BYTES = "clientCertBytes";
+    public static final String CLIENT_KEY_BYTES = "clientKeyBytes";
```

Comment:
Looking at the code change, Endpoint being a default (package-private) scoped class means that these constants are not visible to all the code that uses them and we still have hard-coded strings. Maybe these constants should actually live on NetworkConfig (which sets them), where they would be publicly visible.

We should be using these constants everywhere, including HFCAClient and the unit tests.

Reply:
I agree. Moving the constants to NetworkConfig makes more sense.